### ISSUE TITLE
Add 'aria-pressed' attribute for toggle button

### ISSUE THREAD

**Issue by shufflerAbhi** [Semantic-Org/Semantic-UI-React#3329]
## Feature Request
Add `aria-pressed` attribute for toggle button. 
### Problem description
Accessibility guidelines suggest that toggle buttons have an additional `aria-pressed` attribute. This ensures that screen readers understand that the button supports being pressed and not-pressed.
[MDN guide on accessible toggle buttons ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Toggle_buttons)

### Proposed solution
The `aria-pressed` attribute can be set based on the value of `toggle` and `active` props. If `toggle` is true, the value of `active` can be used to set `aria-pressed` to the corresponding value. If `toggle` is false or undefined, then `aria-pressed` should be undefined.

**Comment by layershifter** [comment]
Fixed in #3330
